.github/ISSUE_TEMPLATE/2_feature_request.yml

```diff
@@ -0,0 +1,26 @@
+name: 🆕 Suggest an improvement or new feature
+description: ___
+labels: enhancement, under discussion
+body:
+  - type: checkboxes
```

Comment:
No need for this checkbox. We do need the link though. Check the original template again

Reply:
Ok, added the link (it had missed my attention) and made it `markdown` .